docs/index.rst

```diff
@@ -6,50 +6,27 @@ Scripture Burrito Documentation
 
 .. image:: ../logo/burrito_logo.png
 
-This is the official documentation for the evolving Scripture Burrito standard. See the development `milestones here <https://github.com/bible-technology/scripture-burrito/milestones>`_).
+After several years of development and testing, we are pleased to announce the availability of **Scripture Burrito 1.0-rc** specification! We recommend that developers of Scripture and Scripture-related applications test and adopt this specification for interchanging data with other systems. Out of the box Scripture Burrito is designed to support the following types of data:
 
-**This is 1.0.0-beta2** based on `JSON Schema <https://json-schema.org/>`_.
+- :ref:`scripture_text_flavor`
+- :ref:`scripture_print_flavor`
+- :ref:`scripture_audio_flavor`
+- :ref:`scripture_sign_language_flavor`
+- :ref:`scripture_braille_flavor`
+- :ref:`scriptural_text_stories_flavor`
 
-Feedback
---------
+We fully expect that many more flavors will need to be developed. We have provided instructions and examples for Extending Scripture Burrito by testing and implementing new flavors. As new flavors are needed by multiple applications we will consider adding them as official flavors in new schema releases.
 
```

Comment:
> We fully expect that many more flavors will need to be developed

I wouldn't want to say this.  The goal is interoperability, not as many flavors as possible, and it's quite likely that a relatively small number of flavors is sufficient.    I think we should say:

1. We are trying to create the flavors needed for common interchange scenarios, and 
2. Extensibility gives people a way to create x- flavors for things that are not in the spec, and 
3. x- flavors that are used for exchange between independent implementations can be considered for addition to the standard.